\hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1UnrewindableBodyError}{}\doxysection{pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Unrewindable\+Body\+Error Class Reference}
\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1UnrewindableBodyError}\index{pip.\_vendor.urllib3.exceptions.UnrewindableBodyError@{pip.\_vendor.urllib3.exceptions.UnrewindableBodyError}}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}urllib3 encountered an error when trying to rewind a body\end{DoxyVerb}
 Inheritance diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Unrewindable\+Body\+Error\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=3.000000cm]{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1UnrewindableBodyError}
\end{center}
\end{figure}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/pip/\+\_\+vendor/urllib3/exceptions.\+py\end{DoxyCompactItemize}
